lighthouse-core/lib/i18n/README.md

```diff
@@ -213,13 +213,15 @@ CTC is a name that is distinct and identifies this as the Chrome translation for
 
 # Appendix
 
+TODO(bckenny): update
```

Comment:
is this TODO as part of this PR or later? :)

Reply:
> is this TODO as part of this PR or later? :)

part of this PR :) I'll try to figure out what I thought should be added